types/puppeteer.d.ts

```diff
@@ -0,0 +1,16 @@
+import puppeteer from 'puppeteer';
+import puppeteerCore from 'puppeteer-core';
+
+/**
+ * @fileoverview Lighthouse should be compatible with puppeteer and puppeteer-core even though the types can be slightly different between the two packages.
+ * Anytime we want to use a Puppeteer type within Lighthouse, we should pull the union type from here rather than one of the packages directly.
```

Comment:
this is probably a good candidate for pulling in as imported typedefs, but we can keep the global for now (someday it'll be ergonomic...)